Web/BootStrap-Assignment2/Search.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>Search Movie</title>
+<link rel="stylesheet" href="css/bootstrap.css" type="text/css">
+<link rel="stylesheet" href="css/ques1.css" type="text/css">
+<script src="js/jquery-1.11.3.js"></script>
+<script src="js/bootstrap.min.js"></script>
+<script type="text/javascript" src="js/main.js"></script>
+<script type="text/javascript" src="js/search.js"></script>
+</head>
+<body>
+	<nav class="navbar navbar-inverse" role="navigation">
+
+		<div class="navbar-header">
+			<button type="button" class="navbar-toggle visible-xs"
+				data-toggle="collapse" data-target="#example-navbar-collapse">
+				<span class="icon-bar"></span> <span class="icon-bar"></span> <span
+					class="icon-bar"></span>
+			</button>
+			<a class="navbar-brand" href="#">Metacube</a>
+		</div>
+		<div class="collapse navbar-collapse" id="example-navbar-collapse">
+			<ul class="nav navbar-nav">
+				<li class="active"><a href="#" class="visible-sm"><span
+						class="glyphicon glyphicon-home"></span></a><a href="Question3.html"
+					class="hidden-sm">Home</a></li>
+				<li><a href="#" class="visible-sm"><span
+						class="glyphicon glyphicon-user"></span></a><a href="#"
+					class="hidden-sm">About</a></li>
```

Comment:
Add more record on JSON and search then every time it's showing only one record. it should show all the matching record not one record